core/plugins/org.polarsys.capella.core.sirius.analysis/src/org/polarsys/capella/core/sirius/analysis/TitleBlockServices.java

```diff
@@ -456,16 +457,24 @@ public void deleteTitleBlock(DAnnotation titleBlock, DDiagram diagram) {
   /**
    * this function removes a line of the Title Block
    * 
-   * @param titleBlockLine:
-   *          the annotation line of a title block
+   * @param cell:
+   *          removed the entire line that contains the annotation cell

```

Comment:
this is an explanation of what the method does, not for the parameter (?)

Reply:
OK, will update.

Reply:
Done